File: osb-core-main/Platform/Integration/Osb.Core.Platform.Integration.Entity/FitBank/Models/Card.cs
using Osb.Core.Platform.Common.Entity.Enums;

namespace Osb.Core.Platform.Integration.Entity.FitBank.Models
{
    public class Card
    {
        public string FullName { get; set; }
        public string IdentifierCard { get; set; }
        public string PanLastDigits { get; set; }
        public bool IsBlocked { get; set; }
        public CardStatus Status { get; set; }
    }
}

File: src/features/card/redux/models/card.ts
import { ReasonCode } from 'features/card/redux/models/reasonCodeEnum'
import { Gender, MaritalStatus } from './Enum'
export interface Card {
  identifierCard: string
  toTaxId: string
  fullName: string
  status: number
  panLastDigits: number
  isBlocked: boolean
  flagName: string
  currentPin?: string
  pin?: string
  confirmationPin?: string
  reasonCode?: ReasonCode
  gender?: Gender
  nationality?: string
  motherName?: string
  birthDate?: string
  maritalStatus?: MaritalStatus
}
